nubis/puppet/files/install-dependencies

```diff
@@ -5,52 +5,18 @@
 
 CPANM="cpanm --configure-timeout 900 --verbose"
 
-for pkg in \
-   Module::Metadata@1.000033 \
-   DateTime::TimeZone@1.64 \
-   Math::Random::ISAAC \
-   File::MimeInfo::Magic \
-   File::Flock@2013.06 \
-   Daemon::Generic \
-   Crypt::OpenPGP \
-   Crypt::SMIME \
-   TheSchwartz \
-   Any::URI::Escape \
-   Chart::Base \
-   HTTP::Lite \
-   JSON::RPC \
-   PatchReader \
-   Authen::Radius \
-   Template::Plugin::GD \
-   Math::BigInt \
-   ElasticSearch@0.68 \
-   Test::Number::Delta \
-   Math::Random::MT \
-   Devel::StackTrace \
-   Convert::Base32 \
-   Auth::GoogleAuth \
-   Net::SFTP \
-   File::Copy::Recursive \
-   MIME::Tools@5.502 \
-   Email::Simple@2.102 \
-   Email::Send@2.198 \
-   Email::MIME::ContentType@1.011 \
-   Email::MIME@1.911 \
-   Email::MIME::Attachment::Stripper@1.316 \
-   Email::Reply@1.202 \
-   Moo~2.000 \
-  ; do
-  echo "Installing $pkg"
-  $CPANM $pkg < /dev/null
-done
+wget -q -O/vendor.tar.gz https://s3.amazonaws.com/moz-devservices-bmocartons/bmo/vendor.tar.gz
```

Comment:
This is sort of scary https://s3.amazonaws.com/moz-devservices-bmocartons/bmo/vendor.tar.gz

No versionning whatsoever in there, so this means that 2 builds from the same git revision could possibly end up with different included data. Generally scary parctice.